lib/filterx/expr-getattr.c

```diff
@@ -52,6 +52,30 @@ _eval(FilterXExpr *s)
   return attr;
 }
 
+static gboolean
+_unset(FilterXExpr *s)
+{
+  FilterXGetAttr *self = (FilterXGetAttr *) s;
+
+  gboolean result = FALSE;
+
+  FilterXObject *variable = filterx_expr_eval_typed(self->operand);
+  if (!variable)
+    return FALSE;
+
+  if (variable->readonly)
```

Comment:
we don't have readonly yet at this point in the history. That part should probably be bubbled up to the next patch.

Reply:
I've fixed this up with a forced push to this branch.